kalite/updates/management/commands/videodownload.py

```diff
@@ -216,6 +216,14 @@ def youtube_dl_cb(stats, progress_callback, *args, **kwargs):
                 "num_total_videos": len(handled_youtube_ids) + len(failed_youtube_ids),
             })
             caching.initialize_content_caches()
+            
+        except TypeError:
+            failed_youtube_ids.append(video.youtube_id)
+            logging.info("TypeError due to internet disconnect")
```

Comment:
I think it's better to just use the string-converted error message, as that might give more detail than assuming what the error could be. See how to use `str` with `BaseException`: https://docs.python.org/2/library/exceptions.html#exceptions.BaseException.